cubeb-api/src/stream.rs

```diff
@@ -3,61 +3,6 @@
 // This program is made available under an ISC-style license.  See the
 // accompanying file LICENSE for details.
 
-//! Stream Functions
```

Comment:
I have moved this doc block down, to the struct.
Mod level docs don't show up if you employ `use stream::*` in the `lib.rs`